tf/examples/EMI-RNN/helpermethods.py

```diff
@@ -1,70 +1,73 @@
 # creates x_train, y_train, x_test, y_test, x_val, y_val npy files. val set is created after cutting out a user provided fraction from train set.
```

Comment:
Can you please remove this line, add a proper comment and our Privacy policy line here ?

Reply:
@metastableB done